content/exchange-guide.md

```diff
@@ -0,0 +1,575 @@
+Alpha Exchange wants to list BTC/XRP and XRP/USD trading pairs partially because listing these pairs will benefit its users. Specifically, this support will allow its users to:
+
+* Deposit XRP _to_ Alpha Exchange _from_ the RCL
+
+* Withdraw XRP _from_ Alpha Exchange _to_ the RCL
+
+* Trade XRP with other currencies, such as BTC, USD, amongst others
+
+## Prerequisites for Supporting XRP
+
+To support XRP, Alpha Exchange must:
+
+* Create and maintain new [accounts](#accounts)
+
+* Create and maintain [balance sheets](#balance-sheets)
+
+See also:
+
+* [Gateway Compliance](https://ripple.com/build/gateway-guide/#gateway-compliance) — Gateways and exchanges are different, but exchanges should also ensure that they are complying with local regulations and reporting to the appropriate agencies.
+
+* [Requirements for Sending to RCL](https://ripple.com/build/gateway-guide/#requirements-for-sending-to-rcl)
+
+* [Requirements for Receiving from RCL](https://ripple.com/build/gateway-guide/#requirements-for-receiving-from-rcl)
+
+* [Gateway Precautions](https://ripple.com/build/gateway-guide/#precautions)
+
+### Accounts
+
+XRP is held in *accounts* (sometimes referred to as *wallets* ) on the Ripple Consensus Ledger (RCL). Accounts on the RCL are different than accounts on other blockchain ledgers, such as Bitcoin, where accounts incur little to no overhead. To submit transactions (for example, [OfferCreate](https://ripple.com/build/transactions/#offercreate) and others used for trading), RCL accounts require XRP [reserves](https://ripple.com/build/reserves/) to protect the ledger against spam and malicious usage. On other blockchains, balances are derived from the previous block. On the RCL, [account objects](https://ripple.com/build/ledger-format/#accountroot) describe several other properties of the account in addition to balances, so accounts are represented in each ledger and can never be destroyed or removed. Exchanges do not need to create accounts for each customer that holds XRP; they can store all of their customers’ XRP in just a few RCL accounts. For more information about RCL accounts, see the [Accounts](https://ripple.com/build/accounts/) article.
+
+To comply with Ripple's recommend best practices, Alpha Exchange should create at least two new [accounts](https://ripple.com/build/accounts/) on the RCL. To minimize the risks associated with a compromised secret key, Ripple recommends creating [_issuing_, _operational_, and _standby_ accounts](https://ripple.com/build/issuing-operational-addresses/) (these are sometimes referred to, respectively, as cold, hot, and warm wallets). The operational/standby/issuing model is intended to balance security and convenience. Exchanges listing XRP should create the following accounts:
+
+* An [_issuing_ account](https://ripple.com/build/issuing-operational-addresses/#issuing-address) to securely hold the majority of XRP and customers' funds. To provide optimal security, this account should be offline.
```

Comment:
> this account should be offline.

This wording is confusing. What we really mean is:

> this account's secret key should be stored offline

Also, another key role of a cold wallet in the exchange use case is that it's the main address where customers send XRP if they want to deposit from the RCL into your exchange. Customers _could_ send to your hot wallets, but hot wallets are disposable, and you don't want customers to have to update their settings every time you dispose of one. You can receive XRP at your cold wallet without taking any action (that is, while the key is offline) so it's always safe for your customers to send to your cold wallet. (It's slightly less convenient in some cases because it means you'll have to refill your warm/hot wallets more often than if those wallets were the ones receiving deposits, but most of the time that minor inconvenience is considered worth it.)